doc/release/upcoming_changes/20414.expired.rst

```diff
@@ -0,0 +1,4 @@
+``alen`` and ``asscalar`` removed
+---------------------------------
+
+The derecated ``np.alen`` and ``np.asscalar`` functions were removed.
```

Comment:
Typo:
```suggestion
The deprecated ``np.alen`` and ``np.asscalar`` functions were removed.
```

Reply:
thanks, good catch - fixed. not rerunning CI for a one-letter typo fix.